Add osu! Philippines Nationals 2022-2023 wiki article (#10331)
ppy/osu-wiki: wiki/Tournaments/oPHN/2023_mania/en.md

## Participants (osu!mania 7K)

- ::{ flag=PH }:: [[LS]Tenshi](https://osu.ppy.sh/users/18520056)
- ::{ flag=PH }:: [- Kura -](https://osu.ppy.sh/users/11420405)
- ::{ flag=PH }:: [Sanni](https://osu.ppy.sh/users/3036686)
- ::{ flag=PH }:: [DaMeMeThEiFxD](https://osu.ppy.sh/users/14324153)
- ::{ flag=PH }:: [Arona](https://osu.ppy.sh/users/8748664)
- ::{ flag=PH }:: [DumbFunk](https://osu.ppy.sh/users/16446375)
- ::{ flag=QA }:: [bustinman](https://osu.ppy.sh/users/19089928)
- ::{ flag=PH }:: [[LS]Shark](https://osu.ppy.sh/users/18270641)
- ::{ flag=PH }:: [bojii the rock](https://osu.ppy.sh/users/10083439)
- ::{ flag=PH }:: [arcwinolivirus](https://osu.ppy.sh/users/2039089)
- ::{ flag=PH }:: [-VRX-](https://osu.ppy.sh/users/8597906)
- ::{ flag=PH }:: [Wewoj](https://osu.ppy.sh/users/3166596)
- ::{ flag=PH }:: [MapleSyrup-](https://osu.ppy.sh/users/1192936)
- ::{ flag=PH }:: [mama mo](https://osu.ppy.sh/users/9085137)
- ::{ flag=PH }:: [bananacake304](https://osu.ppy.sh/users/3741704)
- ::{ flag=PH }:: [ManiaDegengod](https://osu.ppy.sh/users/13193798)
- ::{ flag=AU }:: [shorts](https://osu.ppy.sh/users/8021372)
- ::{ flag=PH }:: [shadowkun-](https://osu.ppy.sh/users/13450837)

> ## Poduim

Reviewer: ```suggestion
## Podium
```

This competition has come to an end and resulted in the following podium:

### osu!mania 4K

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=PH }:: [bojii the rock](https://osu.ppy.sh/users/10083439) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [[KN]CuB-03](https://osu.ppy.sh/users/18560307) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=PH }:: [iid3rp](https://osu.ppy.sh/users/23274559) |

### osu!mania 7K

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=PH }:: [Sanni](https://osu.ppy.sh/users/3036686) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [Arona](https://osu.ppy.sh/users/8748664) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=PH }:: [bojii the rock](https://osu.ppy.sh/users/10083439) |

## Mappools

### 4K

> #### Grand finals

Reviewer: ```suggestion
#### Grand Finals
```

  1. [Sakuraburst - Aethersong (RuleBlazing) [arccat edit]](https://osu.ppy.sh/beatmapsets/1201543#mania/4125634)
  2. [Fellsius - Talk (Cut ver.) (Muses) [talk with sv]](https://osu.ppy.sh/beatmapsets/1815218#mania/3723860)
- Tiebreaker
  1. **[Camellia - Heart of Android : Even If It's Only By Mechanism (AlexDunk) [Machinery Love]](https://osu.ppy.sh/beatmapsets/1072415#mania/2582879)**

#### Qualifiers

- Stage
  1. (SV1) [rintaro soma - solips (notapplicable) [Stage 1: Awakening]](https://osu.ppy.sh/beatmapsets/1996132#mania/4148531)
  2. (RC1) [An - Cochma (Cut Ver.) (TheFunk) [Stage 2: Revert]](https://osu.ppy.sh/beatmapsets/1996135#mania/4148534)
  3. (LN1) [sakuzyo - Imprinting (FelixSpade) [[4K] Stage 3: Insight]](https://osu.ppy.sh/beatmapsets/1995167#mania/4148570)
  4. (RC2) [Akiri vs. HowToPlayLN - Kuroi Hanabira (Lott) [Stage 4: Withered]](https://osu.ppy.sh/beatmapsets/1996162#mania/4148596)
  5. (LN2) [M2U x Gentle Stick - Ineffabilis (Sped Up Ver.) (ERA Hatsuki) [Stage 5: Hatsu-cat's Impulse]](https://osu.ppy.sh/beatmapsets/1995268#mania/4146554)
  6. (HB1) [celtix - Sublime (Saemitsu) [Exalted 1.05x]](https://osu.ppy.sh/beatmapsets/1996147#mania/4148565)

### 7K

> #### Grand finals

Reviewer: ```suggestion
#### Grand Finals
```

  3. [44teru-k - Piano Concerto No. 1 (Makiba) [Roko's Concerto]](https://osu.ppy.sh/beatmapsets/1770177#mania/3623737)
- Tiebreaker
  1. **[wotoha - O108ROCKET feat. Neko Hacker (SurfChu85) [GO TO SPACE!!!]](https://osu.ppy.sh/beatmapsets/2000681#mania/4159728)**

#### Qualifiers

- Stage
  1. (RC1) [The Flashbulb - Warm Hands In Cold Fog (tyrcs) [Stage 1: Warm Up]](https://osu.ppy.sh/beatmapsets/1996198#mania/4148671)
  2. (LN1) [ARForest - Clock Atelier (SurfChu85) [Stage 2: Chronosphere]](https://osu.ppy.sh/beatmapsets/1996130#mania/4148528)
  3. (RC2) [Alestorm - P.A.R.T.Y. (Cut Ver.) (MadBricktree) [Stage 3: Chantey]](https://osu.ppy.sh/beatmapsets/1996212#mania/4148696)
  4. (LN2) [seatrus - Raindrop (Maxus) [Stage 4 : Precipitation]](https://osu.ppy.sh/beatmapsets/1996154#mania/4148579)
  5. (RC3) [Mayuri Shiina - Stardust Apoptosis (Makiba) [Stage 5: Interstellar]](https://osu.ppy.sh/beatmapsets/1996274#mania/4148886)
  6. (HB1) [satie - Skovla (paulkappa) [Stage 6: Forest]](https://osu.ppy.sh/beatmapsets/1996156#mania/4148581)

## Match results

### 4K

> #### Grand finals

Reviewer: ```suggestion
#### Grand Finals
```

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **bojii the rock** ::{ flag=PH }:: | **7** | 1 | ::{ flag=PH }:: [KN]CuB-03 | [#1](https://osu.ppy.sh/community/matches/109423478) |
| [KN]CuB-03 ::{ flag=PH }:: | 2 | **7** | ::{ flag=PH }:: **bojii the rock** | [#1](https://osu.ppy.sh/community/matches/109422864) |
| **bojii the rock** ::{ flag=PH }:: | **7** | 2 | ::{ flag=PH }:: iid3rp | [#1](https://osu.ppy.sh/community/matches/109389834) |

#### Finals

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| iid3rp ::{ flag=PH }:: | 4 | **7** | ::{ flag=PH }:: **[KN]CuB-03** | [#1](https://osu.ppy.sh/community/matches/109294100) |
| **bojii the rock** ::{ flag=PH }:: | **7** | 0 | ::{ flag=PH }:: [KN]Puddles | *win by default* |
| Vulpsi ::{ flag=PH }:: | 0 | **7** | ::{ flag=PH }:: **[KN]Puddles** | *win by default* |
| Dyei ::{ flag=PH }:: | 0 | **7** | ::{ flag=PH }:: **bojii the rock** | [#1](https://osu.ppy.sh/community/matches/109278242) |

> #### Semiinals

Reviewer: ```suggestion
#### Semifinals
```

| **bojii the rock** ::{ flag=PH }:: | **6** | 0 | ::{ flag=PH }:: TigerWasTaken | *win by default* |

#### Round of 16

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Anosious** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: [LS]Tenshi | *win by default* |
| **Vulpsi** ::{ flag=PH }:: | **5** | 3 | ::{ flag=PH }:: ManiaDegengod | [#1](https://osu.ppy.sh/community/matches/108948515) |
| **Silicosis** ::{ flag=PH }:: | **5** | 3 | ::{ flag=PH }:: Xyrenoxx | [#1](https://osu.ppy.sh/community/matches/108950762) |
| **[KN]CuB-03** ::{ flag=PH }:: | **5** | 1 | ::{ flag=PH }:: DiamondGenius75 | [#1](https://osu.ppy.sh/community/matches/108928105) |
| **[KN]Puddles** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: Sailoring | [#1](https://osu.ppy.sh/community/matches/108946868) |
| **iid3rp** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: - Kura - | *win by default* |
| **Dyei** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: TigerWasTaken | *win by default* |
| bojii the rock ::{ flag=PH }:: | 0 | **5** | ::{ flag=PH }:: **Axfaerie** | *win by default* |

### 7K

> #### Grand finals

Reviewer: ```suggestion
#### Grand Finals
```

| DaMeMeThEiFxD ::{ flag=PH }:: | 2 | **6** | ::{ flag=PH }:: **shadowkun-** | [#1](https://osu.ppy.sh/community/matches/109063116) |

#### Round of 16

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **bojii the rock** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: DumbFunk | *win by default* |
| **arcwinolivirus** ::{ flag=PH }:: | **5** | 1 | ::{ flag=QA }:: bustinman | [#1](https://osu.ppy.sh/community/matches/108950544) |
| **mama mo** ::{ flag=PH }:: | **5** | 4 | ::{ flag=PH }:: MapleSyrup- | [#1](https://osu.ppy.sh/community/matches/108951465) |
| **Sanni** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: bananacake304 | [#1](https://osu.ppy.sh/community/matches/108929190) |
| **Wewoj** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: [LS]Tenshi | *win by default* |
| - Kura - ::{ flag=PH }:: | 0 | **5** | ::{ flag=PH }:: **[LS]Shark** | *win by default* |
| shadowkun- ::{ flag=PH }:: | 3 | **5** | ::{ flag=PH }:: **-VRX-** | [#1](https://osu.ppy.sh/community/matches/108950982) |
| **Arona** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: DaMeMeThEiFxD | [#1](https://osu.ppy.sh/community/matches/108929237) |

## Ruleset

> ### Tournament Organization

Reviewer 2: Except for headers saying 'Grand Finals' I should add.

- One player has achieved enough wins in such a way that the other player cannot successfully win the round.
  - The winning player moves onto the next match.
  - If the match was in the winner's bracket or during the first round, the losing player falls to the losers bracket. If a player loses in the loser’s bracket, they are eliminated from the tournament.
- Both players are at match point at the same time.
  - Both players will play the song designated as the tiebreaker.
  - The winner of the tiebreaker will progress to the next match of the tournament, while the loser will be eliminated or sent to the losers bracket, depending on the position of the tournament.

Players will have 2 minutes to pick a map and 2 minutes to get ready. Players also allowed to pick their opponent's protect.

- In case a player takes more than 2 minutes to pick a map, they will receive a warning from the referee, on subsequent occurrences the referee will pick a random map available to pick from the mappool using !roll X, where X is the number of available maps to pick in the mappool.

> - Each player is entitled to an optional (not required to be used) time out of two minutes when its their turn to pick. This time out period will only be allowed once per player per match.

Reviewer 2: ```suggestion
- Each player is entitled to an optional (not required to be used) timeout of two minutes when its their turn to pick. This timeout period will only be allowed once per player per match.
```

#### Disconnections

Maps with a player disconnection that occurs within the first 30 seconds, or 25% of the song (whichever comes first) will be nullified and re-attempted.

Matches will have a two disconnection limit - upon the third disconnection, a point will be handed immediately to the player still active in the multiplayer room.

> A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the 10 minute grace period mentioned in Section 2.1 (Protocol) will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.

Reviewer 2: ```suggestion
A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the 10 minute grace period mentioned in [match protocol](#match-protocol) will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.
```

- ::{ flag=PH }:: [ManiaDegengod](https://osu.ppy.sh/users/13193798)
- ::{ flag=AU }:: [shorts](https://osu.ppy.sh/users/8021372)
- ::{ flag=PH }:: [shadowkun-](https://osu.ppy.sh/users/13450837)

## Podium

This competition has come to an end and resulted in the following podium:

### osu!mania 4K

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=PH }:: [bojii the rock](https://osu.ppy.sh/users/10083439) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [[KN]CuB-03](https://osu.ppy.sh/users/18560307) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=PH }:: [iid3rp](https://osu.ppy.sh/users/23274559) |

### osu!mania 7K

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=PH }:: [Sanni](https://osu.ppy.sh/users/3036686) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [Arona](https://osu.ppy.sh/users/8748664) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=PH }:: [bojii the rock](https://osu.ppy.sh/users/10083439) |

## Mappools

### 4K

> #### Grand finals

Reviewer 2: ```suggestion
#### Grand Finals
```
While headers are sentence case, the term 'Grand Finals' should have both Grand and Finals capitalised.

| DaMeMeThEiFxD ::{ flag=PH }:: | 2 | **6** | ::{ flag=PH }:: **shadowkun-** | [#1](https://osu.ppy.sh/community/matches/109063116) |

#### Round of 16

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **bojii the rock** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: DumbFunk | *win by default* |
| **arcwinolivirus** ::{ flag=PH }:: | **5** | 1 | ::{ flag=QA }:: bustinman | [#1](https://osu.ppy.sh/community/matches/108950544) |
| **mama mo** ::{ flag=PH }:: | **5** | 4 | ::{ flag=PH }:: MapleSyrup- | [#1](https://osu.ppy.sh/community/matches/108951465) |
| **Sanni** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: bananacake304 | [#1](https://osu.ppy.sh/community/matches/108929190) |
| **Wewoj** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: [LS]Tenshi | *win by default* |
| - Kura - ::{ flag=PH }:: | 0 | **5** | ::{ flag=PH }:: **[LS]Shark** | *win by default* |
| shadowkun- ::{ flag=PH }:: | 3 | **5** | ::{ flag=PH }:: **-VRX-** | [#1](https://osu.ppy.sh/community/matches/108950982) |
| **Arona** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: DaMeMeThEiFxD | [#1](https://osu.ppy.sh/community/matches/108929237) |

## Ruleset

> ### Tournament organization

Reviewer 2: ```suggestion
### Tournament organisation
```

#### Qualification process

All players that enter the tournament will be eligible to play in the qualification round. However, the number of participants who will advance to the next round will be cut down to the nearest power of 2. The maximum number of players who can advance to the bracket stage is 32. The criterion used to determine whether a player will progress to the next round is their performance in the qualification stage.

The qualification round will have a 6-song mappool that widely varies in skillsets tested. These tested skillsets will correspond to skillsets commonly seen in mappools throughout the tournament.

**All six notecharts will be created by the mappool selectors and/or special guest notecharters and will be unreleased prior to the tournament qualification stage, to provide zero opportunity for advantage through familiarity.**

All players will have two chances at each qualifier notechart, in which the highest of the two runs on each qualifier notechart will be used to calculate your final score.

> Players will be organized into groups based on their availability. Because qualification is not a process that involves 1 on 1 matchups, it is actually possible to have 8 players go through the qualification process at once. If this is a confusing process, do not worry: simply adhere to the schedule that will be released and follow referee instructions during the match.

Reviewer 2: ```suggestion
Players will be organised into groups based on their availability. Because qualification is not a process that involves 1 on 1 matchups, it is actually possible to have 8 players go through the qualification process at once. If this is a confusing process, do not worry: simply adhere to the schedule that will be released and follow referee instructions during the match.
```
